src/pages/currency-trading/forms/TransactionForm.js

```diff
@@ -484,29 +445,26 @@ export default function TransactionForm({ recordId, labels, maxAccess, plantId,
   }
   }
 
-  const total = formik.values.rows.reduce((acc, { lcAmount }) => {
-    // Convert lcAmount to string and replace commas
+  const total = formik.values.operations.reduce((acc, { lcAmount }) => {
     const amountString = String(lcAmount || 0).replaceAll(',', '')
 
-    // Parse the amount and add to accumulator
     return acc + parseFloat(amountString) || 0
   }, 0)
 
-  const receivedTotal = formik.values.rows2.reduce((acc, { amount }) => {
-    // Convert lcAmount to string and replace commas
+  const receivedTotal = formik.values.amount.reduce((acc, { amount }) => {
     const amountString = String(amount || 0).replaceAll(',', '')
```

Comment:
We're still parsing formatting here. Those should be inside numberfield. No handling number as text in this file.